rasa/core/broker.py

```diff
@@ -1,6 +1,10 @@
 import json
 import logging
-from typing import Any, Dict, Optional, Text
+import os
+import ssl
+from typing import Any, Dict, Optional, Text, Union
+
+import pika
```

Comment:
optional: there are a couple of other `import pika`s down below. Either globally or locally.

Reply:
they're all local now actually